File: app/src/main/java/com/ogh/support/bean/WenZhangBean.java
package com.ogh.support.bean;

import com.ogh.frame.bean.BaseBean;

import java.util.List;

/**
 * @describe 文章实体类
 */
public class WenZhangBean extends BaseBean {

    public DataBean data;

    public static class DataBean {

        public List<DatasBean> datas;

        public static class DatasBean {

            public String title;
        }
    }

    @Override
    public boolean isEmpty() {
        return null == data.datas || data.datas.isEmpty();
    }
}

File: app/src/main/java/com/ogh/support/bean/WenZhangBean.kt
package com.ogh.support.bean

import com.ogh.frame.bean.BaseBean


/**
 * @describe 文章实体类
 */
data class WenZhangBean(val data: DataBean) : BaseBean() {

    data class DataBean(val datas: List<DatasBean>) {
        data class DatasBean(val title: String)
    }

    override fun isEmpty(): Boolean {
        return data.datas.isEmpty()
    }
}